apps/docs/app/features/portable-text/PortableText.tsx

```diff
@@ -264,7 +264,7 @@ const components: Partial<PortableTextReactComponents> = {
     tipsPanel: ({ value }) => (
       <Box as="article" backgroundColor="mint" mt={3} p={4} borderRadius="md">
         <Flex gap={1} alignItems="end">
-          <FavouriteOutline30Icon />
+          <FavouriteOutline30Icon aria-hidden="true" />
```

Comment:
Hvordan foretrekker du aria-hidden="true" vs. kun aria-hidden, for ettersom man ikke tar den med i det hele tatt, så er den vel false? Men aria-hidden="true" føles kanskje mer eksplisitt?

Reply:
Egentlig ikke noe poeng å ha det nei. Og med endringen ved at dette er defaulten uansett, så kan jeg vel fjerne hele sulamitten.